cmd/test-bot.rb

```diff
@@ -73,15 +73,15 @@ def test_bot_args
              description: "Only run the formulae dependents steps."
       switch "--only-cleanup-after",
              description: "Only run the post-cleanup step. Needs `--cleanup`."
-      flag   "--testing-formulae=",
-             description: "Use these testing formulae rather than running the formulae detection steps."
-      flag   "--added-formulae=",
-             description: "Use these added formulae rather than running the formulae detection steps."
-      flag   "--deleted-formulae=",
-             description: "Use these deleted formulae rather than running the formulae detection steps."
-      flag   "--skipped-or-failed-formulae=",
-             depends_on:  "--only-formulae-dependents",
```

Comment:
Is this intentionally being removed? It looks to me like the only time this is used is if the `formulae-dependents` test runs without the `formulae` test running. Which I believe only happens if passing `--only-formulae-dependents`.

Reply:
It's not supported for `comma_array` currently.

Reply:
Ah well in that case, 👍 😅